README.md

```diff
@@ -0,0 +1,155 @@
+<a name="readme-top"></a>
+### React App
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#In this project we build simple Book list application that can add and remove the Books)
+  - [🛠 Built With](#build-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [React-app] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[React-app]** In this project we build simple application that have ability to add and remove books
+
```

Comment:
- [ ] Kindly fix your project title and description.

- [ ] Kindly remove the text from the template, "Describe your project in 1 or 2 sentences.". This is for demo purposes and should not be a part of the update readme.